pkg/storaged/stratis-details.jsx

```diff
@@ -291,12 +291,16 @@ export const StratisPoolDetails = ({ client, pool }) => {
     function set_mount_options(path, vals) {
         let mount_options = [];
 
-        if (!vals.mount_options.auto || vals.mount_options.never_auto)
+        if (vals.variant == "nomount" || vals.at_boot == "never")
             mount_options.push("noauto");
         if (vals.mount_options.ro)
             mount_options.push("ro");
-        if (vals.mount_options.never_auto)
+        if (vals.at_boot == "never")
```

Comment:
This added line is not executed by any test.  [Details](https://cockpit-logs.us-east-1.linodeobjects.com/pull-17888-20221115-130151-a219e579-fedora-36-devel//Coverage/lcov/github-pr.diff.gcov.html)